app/presenters/content_item/brexit_taxons.rb

```diff
@@ -8,15 +8,15 @@ module BrexitHubPage
 
     def brexit_links
       {
-        ContentItem::BrexitHubPage::BREXIT_BUSINESS_PAGE_CONTENT_ID => {
+        ContentItem::BrexitTaxons::BREXIT_BUSINESS_PAGE_CONTENT_ID => {
```

Comment:
We should be able to get rid of `ContentItem::BrexitTaxons::` in this bit as it's referring to things within this class, shouldn't we? We'll obvs need to keep them in the tests